ui/e2e/src/tests/markdown.spec.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import happoPlaywright from 'happo-playwright';
+import { test } from '../fixtures/dashboardTest';
+
+test.use({
+  dashboardName: 'MarkdownPanel',
+});
+
+test.describe('Dashboard: Markdown Panel', () => {
+  test.beforeEach(async ({ context }) => {
+    await happoPlaywright.init(context);
+  });
+
+  test.afterEach(async () => {
+    await happoPlaywright.finish();
+  });
+
+  ['Headings', 'Text', 'Links', 'Code', 'Lists', 'Tables'].forEach((name) => {
+    test(`displays ${name} as expected`, async ({ page, dashboardPage }) => {
+      const markdownPanel = dashboardPage.getPanel(name);
+      await markdownPanel.isLoaded();
+
+      await happoPlaywright.screenshot(page, markdownPanel.container, {
+        component: 'Markdown Panel',
+        variant: `${name} [light]`,
+      });
```

Comment:
I guess we're always guaranteed to be in light mode when we start things from scratch? (Wondering if local storage data would ever cause a problem with this)

Reply:
Assuming it's the default behavior in a fresh browser, I _think_ it's safe, but let me poke at that more closely to be sure.

Reply:
Playwright uses a fresh browser for each test and the theme defaults to light mode, so it should be safe unless the underlying behavior changes. I added in an `isLightMode` and `isDarkMode` check just to be safe, so we'll get a meaningful test failure if that underlying behavior changes.